dt/uni04delta/edpm/nodeset/kustomization.yaml

```diff
@@ -189,3 +189,75 @@ replacements:
           - spec.services
         options:
           create: true
+
+  - source:
+      kind: ConfigMap
+      name: service-values
+      fieldPath: data.neutron.customServiceConfig
+    targets:
+      - select:
+          kind: OpenStackControlPlane
+        fieldPaths:
+          - spec.neutron.template.customServiceConfig
+        options:
+          create: true
```

Comment:
I see this was removed from the control plane step and placed into a data plane step instead.  Is that because it wasn't really doing anything during the control plane step and is more properly placed here instead?